PYTHON/VariableLibrary/chat_completion_with_azure_openai.py

```diff
@@ -0,0 +1,91 @@
+# Select 'Manage connections' and add a connection to a Variable Library
+# Replace the alias "<My Variable Library Alias>" with your connection alias.
+from openai import AzureOpenAI
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.secrets import SecretClient
+
+@udf.connection(argName="varLib", alias="<My Variable Library Alias>")
+@udf.function()
+def chat_request(prompt: str, varLib: fn.FabricVariablesClient) -> str:   
+    '''
+    Description: Sends a chat completion request to an Azure OpenAI model using configuration values 
+    retrieved from a Fabric Variable Library and Azure Key Vault.
+    
+    Pre-requisites: 
+        * Create an Azure OpenAI endpoint in Azure Portal
+        * Create an Azure Key Vault and store your Azure OpenAI API key as a secret
+        * Grant your Fabric workspace/app access to read secrets from Key Vault
```

Comment:
Might want to add this link for granting access https://learn.microsoft.com/en-us/fabric/data-factory/azure-key-vault-reference-overview. Create docs are easy to find but this could be hard to find . Might be good to add it in the comments.